content/maintainers-guide/project-organization.md

```diff
@@ -0,0 +1,85 @@
+
+Current product managers:
+
+- Fabien Vallon ([@fabienvallon](https://github.com/fabienvallon))
+- Marion François ([@marionf](https://github.com/marionf))
+- Mateus Shirlaw ([@MatShir](https://github.com/MatShir))
+
+### Maintainer
+
+Maintainers provide the project's technical vision, review code contributions, and ensure that the software develops in line with the product vision.
+
+Current maintainers:
+
+- Franck Lefèvre ([@Progi1984](https://github.com/Progi1984))
+- Ibrahima Sow ([@sowbiba](https://github.com/sowbiba))
+- Jonathan Lelièvre ([@jolelievre](https://github.com/jolelievre))
+- Krystian Podemski ([@kpodemski](https://github.com/kpodemski))
+- Mathieu Ferment ([@matks](https://github.com/matks))
+- Matthieu Rolland ([@matthieu-rolland](https://github.com/matthieu-rolland))
+- Pablo Borowicz ([@eternoendless](https://github.com/eternoendless))
+- Pierre Rambaud ([@PierreRambaud](https://github.com/PierreRambaud))
+- Pululu Kinanga André ([@PululuK](https://github.com/PululuK))
+- Thomas Baccelli ([@atomiix](https://github.com/atomiix))
+- Valentin Szczupak ([@NeOMakinG](https://github.com/NeOMakinG))
+
+### UX Designer
+
+UX Designers make sure that interfaces follow the best practices in usability and design.
+
+Current UX designers:
+
+- Scott Cosio ([@prestascott](https://github.com/prestascott))
+- Tristan Lehot ([@TristanLDD](https://github.com/TristanLDD))
+
+### Quality Assurance (QA)
+
+Issue managers qualify new issues to avoid duplicates and, if they are indeed defects in the software, their severity.
+
+Current issue managers:
+
+- Florine Hea ([@florine2623](https://github.com/florine2623))
+- Hana Rebai ([@HanaRebaiQA](https://github.com/HanaRebaiQA))
+- Hibatallah Aouadni ([@hibatallahAouadni](https://github.com/hibatallahAouadni))
+- Khouloud Belguith ([@khouloudbelguith](https://github.com/khouloudbelguith))
+- Marwa Chelly ([@marwachelly](https://github.com/marwachelly))
+
+QA analysts make sure that the quality of the software improves over time, by verifying that code changes behave as expected.
+
+Current QA analysts:
+
+- Florine Hea ([@florine2623](https://github.com/florine2623))
+- Hana Rebai ([@HanaRebaiQA](https://github.com/HanaRebaiQA))
+- Hibatallah Aouadni ([@hibatallahAouadni](https://github.com/hibatallahAouadni))
+- Khouloud Belguith ([@khouloudbelguith](https://github.com/khouloudbelguith))
+- Marwa Chelly ([@marwachelly](https://github.com/marwachelly))
+- Robin Fischer ([@Robin-Fischer-PS](https://github.com/Robin-Fischer-PS))
+- Sarah Dib ([@sarahdib](https://github.com/sarahdib))
+
+QA automation engineers write automated functional tests, which help accelerate the work of QA analysts.
+
+Current QA automation engineers:
+
+- Nesrine Abdmouleh ([@nesrineabdmouleh](https://github.com/nesrineabdmouleh))
+- Stéphane Decisy ([@SD1982](https://github.com/SD1982))
+- Boubker Bribri ([@boubkerbribri](https://github.com/boubkerbribri))
+
+### Wording Manager
```

Comment:
```suggestion
### Product Content Manager
```

That's the full name 😇

Reply:
Sound pompous for an Open Source project to me.
But if you want to keep 'Product Content Manager' , it's fine

Reply:
"Content Manager" would work too, but "wording manager" sounds a bit simplistic to me. 😇

Reply:
This is role, not job.
Your job is more complexe than the role

For your job ( Product Content Manager ) ,  2 roles for the community 
- Wording  maintener 
- Documentation maintener

Reply:
Can we please settle on a name for this role? We can add more detail after if necessary.